airbyte-workers/src/main/java/io/airbyte/workers/temporal/check/connection/CheckConnectionWorkflowImpl.java

```diff
@@ -18,11 +18,11 @@ public class CheckConnectionWorkflowImpl implements CheckConnectionWorkflow {
       Workflow.newActivityStub(CheckConnectionActivity.class, ActivityConfiguration.CHECK_ACTIVITY_OPTIONS);
 
   @Override
-  public StandardCheckConnectionOutput run(final JobRunConfig jobRunConfig,
-                                           final IntegrationLauncherConfig launcherConfig,
-                                           final StandardCheckConnectionInput connectionConfiguration) {
+  public ConnectorJobOutput run(final JobRunConfig jobRunConfig,
+                                final IntegrationLauncherConfig launcherConfig,
+                                final StandardCheckConnectionInput connectionConfiguration) {
 
-    return activity.run(new CheckConnectionInput(jobRunConfig, launcherConfig, connectionConfiguration));
+    return activity.runWithJobOutput(new CheckConnectionInput(jobRunConfig, launcherConfig, connectionConfiguration));
```

Comment:
it is less likely to happen because of the time that it takes to run this activity but we will need a version here.

Reply:
added in [aefc013](https://github.com/airbytehq/airbyte/pull/14715/commits/aefc0135a6898e23ef6088636c31946bc503c940)